server/src/controllers/song.controllers.ts

```diff
@@ -0,0 +1,106 @@
+import { Request, RequestHandler, Response } from 'express';
+import { Song }  from '../models/song.model';
+import { SongDocument } from '../types/song.types';
+
+const sendResponse = (res: Response, statusCode: number, payload: SongDocument[] | SongDocument | string ) => {
+  return res.status(statusCode).json(payload);
+};
+
+const createSong: RequestHandler = async (req: Request, res: Response): Promise<SongDocument | string> => {
```

Comment:
This function should return nothing, as we are just connecting to the db, we don't need any returned values from this function. Shod be Promise<void>, then u can delete return as tam mentioned. Same thing with the other APIs